### DISCUSSION TITLE
Installed but no css - gulpfile.js runs fine

### DISCUSSION THREAD

**Discussion by Velocedge** [POPWorldMedia/POPForums#262] [General]
I'm running the project in IIS Express VS 2022 with Node.js and NPM.  It builds the database and launches the web app but there is no formatting.  Setup says run Task Runner Explorer on gulpfile.js.  This is the output which looks ok:

D:\Data\CADE.net\POPForums-18.0.0\src\PopForums.Web> cmd.exe /c gulp -b "D:\Data\CADE.net\POPForums-18.0.0\src\PopForums.Web" --color --gulpfile "D:/Data/CADE.net/POPForums-18.0.0/src/PopForums.Web/Gulpfile.js" default
[13:41:09] Using gulpfile D:\Data\CADE.net\POPForums-18.0.0\src\PopForums.Web\Gulpfile.js
[13:41:09] Starting 'default'...
[13:41:09] Starting 'min'...
[13:41:09] Starting 'copies'...
[13:41:09] Finished 'copies' after 59 ms
[13:41:09] Starting 'js'...
[13:41:11] Finished 'js' after 2.15 s
[13:41:11] Starting 'css'...
[13:41:11] Finished 'css' after 92 ms
[13:41:11] Starting 'bump'...
[13:41:11] Bumped 18.0.1-0 to 18.0.1-1 with type: prerelease
[13:41:11] Finished 'bump' after 13 ms
[13:41:11] Finished 'min' after 2.31 s
[13:41:11] Finished 'default' after 2.31 s
Process terminated with code 0.

Not sure where to go next.  Have I missed something?

**Comment by jeffputz** [comment]
Is the file physically there? Is the browser showing it 404? Did you try to shift-refresh the browser?

**Reply by Velocedge** [reply]
I'm not even sure where or what file I am looking for.  There are .css files in PopForums.Web\wwwroot\lib\PopForums\dist and src.  No 404, just no styling on the page.
![image](https://user-images.githubusercontent.com/3296886/157658174-24b30dbd-952a-4224-8136-24e715c61536.png)

**Comment by jeffputz** [comment]
Look at the network tab of the developer tools. Look at the source of the page (there's only one CSS file).

**Reply by Velocedge** [reply]
Should have probably made myself more clear.  I see those css files I referenced. <link href="[/lib/PopForums/src/PopForums.css](https://localhost:44321/lib/PopForums/src/PopForums.css)" rel="stylesheet" /> is the only link in the page.  Question is, if the page is loading PopForums.css, why does the page look like the the image I posted (which appears to not have any css applied)

**Reply by jeffputz** [reply]
I couldn't tell you. Did you change the markup? Right click and inspect an element and see if any CSS is applied.

The CI builds are the same code, and work fine.
